go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -751,6 +752,24 @@ func (tm *TabletManager) findMysqlPort(retryInterval time.Duration) {
 	}
 }
 
+// redoPreparedTransactionsAndSetReadWrite redoes prepared transactions in read-only mode.
+// We turn off super read only mode, and then redo the transactions. Finally, we turn of read-only mode to allow for further traffic.
```

Comment:
// We turn off super read only mode, and then redo the transactions. Finally, we turn of read-only mode to allow for further traffic.

This line is confusing me. `Finally we turn on/off`?